shared/chat/conversation/info-panel/index.js

```diff
@@ -82,13 +83,13 @@ type infoPanelProps = {
   }>,
 }
 
-type SmallTeamInfoPanelProps = infoPanelProps & {
+type ConversationInfoPanelProps = infoPanelProps & {
   onShowBlockConversationDialog: () => void,
   onShowNewTeamDialog: () => void,
   showTeamButton: boolean,
 }
 
-const _SmallTeamInfoPanel = (props: SmallTeamInfoPanelProps) => (
+const _ConversationInfoPanel = (props: ConversationInfoPanelProps) => (
```

Comment:
Copied SmallTeamInfoPanel and renamed it ConversationInfoPanel

Reply:
Ah, got it, we're making a distinction between small teams, big teams, and old-style conversations.